src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/WikiBasePageObject.java

```diff
@@ -54,15 +55,29 @@
 import org.openqa.selenium.WebDriverException;
 import org.openqa.selenium.WebElement;
 import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.FindBys;
 
+import java.util.ArrayList;
 import java.util.List;
 import java.util.concurrent.TimeUnit;
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
+import java.util.stream.Collectors;
 
 
 public class WikiBasePageObject extends BasePageObject {
 
+
+  public static class PageMessages{
+
+    private PageMessages() {
+      throw new IllegalAccessError("Utility class");
+    }
+
+    public static final String INVALID_NUMBER_OF_CONFIRMING_NOTIFICATIONS =
+            "Number of action confirming notifications is invalid";
+    public static final String BANNER_NOTIFICATION_NOT_VISIBLE = "Banner notification message is not visible";
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this variable to comply with Java Code Conventions. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1213)